consensus/istanbul/backend/engine.go

```diff
@@ -198,6 +198,22 @@ func (sb *Backend) verifyCascadingFields(chain consensus.ChainHeaderReader, head
 		if err := sb.verifySigner(chain, header, parents); err != nil {
 			return err
 		}
+
+		if !chain.Config().IsGingerbread(header.Number) {
```

Comment:
Since there also is an else clause, it's a bit nicer to read if you start with the Gingerbread rather than having `if !bool {...} else {...}`.